cnbinfo/readme.rst

```diff
@@ -0,0 +1,17 @@
+Description
+Is a python script that get all the usernames and ids for a given users list.
+First, we need to generate this user list file. You can generate it by running:
+  ./multirun.py --literal 'cat /etc/passwd' > users.txt
+And then you can run  ./extract_users.py without arguments.
+
+Usage
```

Comment:
This is redundant we can probably have one single section, don't you think?

Reply:
Ok. I write Usage now as a bold word